docs/source/basics/chat_datasets.rst

```diff
@@ -0,0 +1,248 @@
+.. _chat_dataset_usage_label:
+
+=============
+Chat Datasets
+=============
+
+Chat datasets involve multi-turn conversations (multiple back-and-forths) between user and assistant.
+
+.. code-block:: python
+
+    [
+        {"role": "user", "content": "What is the answer to the ultimate question of life?"},
+        {"role": "assistant", "content": "The answer is 42."},
+        {"role": "user", "content": "That's ridiculous"},
+        {"role": "assistant", "content": "Oh I know."},
+    ]
+
+This is more structured than freeform text association that models are typically pre-trained with,
+where they learn to simply predict the next token instead of responding accurately to the user.
+
+The primary entry point for fine-tuning with chat datasets in torchtune is the :func:`~torchtune.datasets.chat_dataset`
+builder. This lets you specify a local or Hugging Face dataset that follows the chat data format
+directly from the config and train your LLM on it.
+
+Example chat dataset
+--------------------
+
+.. code-block:: bash
+
+    head data/my_data.json -n 22
+    # [
```

Comment:
hashes for why?

Reply:
probably better to just show the json file directly